src/render/mod.rs

```diff
@@ -130,6 +173,7 @@ impl Plugin for TilemapRenderingPlugin {
 
         let render_app = app.sub_app_mut(RenderApp);
         render_app
+            .insert_resource(ChunkCoordinates::new(chunk_size))
```

Comment:
I would prefer if this said `RenderChunkCoordinates` as chunks are very much an internal thing right now and only a way for the renderer to split large tilemaps into more manageable data sizes.